### ISSUE TITLE
[F-Req] If a server does not have the nqn bot, allow nitrospoof to handle

### ISSUE THREAD

**Issue by Henry-Hiles** [RhythmLunatic/aliucord-plugins#12]
For nitrospoof for nqn, i would like, if the emoji cannot be rendered with nqn, let the regular nitrospoof plugin turn it into a link, or add the ability ti send as a link to this plugin. Thank you.

**Comment by RhythmLunatic** [comment]
There is no way to determine if the original server with the emoji has the NQN bot so this is nearly impossible.
As an alternative, ask your server owners to add NQN or purchase Nitro.

**Comment by RhythmLunatic** [comment]
Never mind I just need to check if the current server has NQN.
I probably won't implement it soon but I will keep it in mind.

**Comment by Henry-Hiles** [comment]
> Never mind I just need to check if the current server has NQN and post links if it doesn't, it can be assumed the original server also has NQN. I probably won't implement it but I will keep it in mind.

Yeah this is really what i wanted. Thank you so much. If you add this this will be the __best__ plugin ever.
